src/providers/LoginProvider.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+import { createContext } from "react";
+
+export const LoginContext = createContext({});
+
+export const LoginProvider = ({ children }) => {
+  const [isLogin, setIsLogin] = useState(false);
```

Comment:
[nits]
Login は名詞なので、isLoggedIn とかの方が良さそうです ☺️ 

```suggestion
  const [isLogin, setIsLogin] = useState(false);
```